### ISSUE TITLE
Profiler error

### ISSUE THREAD

**Issue by TrueCalfa** [screepers/screeps-profiler#25]
Just in case is helpful for you, this is what i get when i do `Game.profiler.reset()`
```
screeps-profiler:214
      if (currentLength + line.length + 1 < outputLengthLimit) {
                              ^
TypeError: Cannot read property 'length' of undefined
    at Object.output (screeps-profiler:214:31)
    at Object.printProfile (screeps-profiler:184:26)
    at Object.report (screeps-profiler:282:16)
    at Object.endTick (screeps-profiler:276:16)
    at Object.wrap (screeps-profiler:326:16)
    at Object.module.exports.loop (main:14:16)
    at __mainLoop:1:52
    at sigintHandlersWrap (vm.js:32:31)
```

**Comment by wtfrank** [comment]
if I do Game.profiler.profile(1), then after 1 tick I get "undefined" then after the second tick I get exactly the same error as above

**Comment by gdborton** [comment]
@TrueCalfa @wtfrank This has been fixed in 2.0.1 - https://github.com/screepers/screeps-profiler/blob/master/CHANGELOG.md
